pkg/frontend/mysql_protocol.go

```diff
@@ -1570,6 +1578,52 @@ func (mp *MysqlProtocolImpl) sendEOFOrOkPacket(warnings, status uint16) error {
 	}
 }
 
+func setColLength(column *MysqlColumn, width int32) {
+	switch column.columnType {
+	case defines.MYSQL_TYPE_DECIMAL:
```

Comment:
We need to have a defines.MYSQL_TYPE type and a GetLength() method.

Reply:
ok